soap/src/java/com/zimbra/soap/mail/message/SendMsgRequest.java

```diff
@@ -105,13 +105,6 @@ public class SendMsgRequest {
     @XmlElement(name=MailConstants.E_MSG /* m */, required=false)
     private MsgToSend msg;
 
-    /**
-     * @zm-api-field-tag delivery-receipt-notification
-     * @zm-api-field-description If set, delivery receipt notification will be sent.
-     */
-    @XmlAttribute(name=MailConstants.A_DELIVERY_RECEIPT_NOTIFICATION /* deliveryReport */, required=false)
```

Comment:
will it break if someone is calling this API? It may not be internal, but if a customer has built anything on top of this, could it cause issues?

Reply:
@viditZim Please include this point in documentation clearly and let @dawoodshaikh know about the impact on changing the api as we discussed earlier